### PR TITLE
Disable stages

### PR DESCRIPTION
**Pull request by zmalik**
This PR is solve the issue commented in #95 

With this way we can do
```
landscaper apply --disable delete files1 
```

```
landscaper apply --disable create --disable delete files1 
```

### CODE REVIEW

**Comment by rollulus** [comment]
@zmalik and @ivanilves , I tried to contact you guys on the Kubernetes Slack without success. Is there another way to get in touch?

**Comment by zmalik** [comment]
sorry @rollulus I just saw the message now and have quickly answered on the Kubernetes Slack

**Comment by ivanilves** [comment]
Hey @rollulus via the GH for example :smiley:

**Comment by zmalik** [comment]
@Fsero can you please give me a code review in this PR?

File: cmd/apply.go [hunk 1]
```diff
@@ -109,6 +109,7 @@ func init() {
 	f.StringVar(&env.HelmHome, "chart-dir", helmHome, "(deprecated; use --helm-home) Helm home directory")
 	f.StringVar(&env.HelmHome, "helm-home", helmHome, "Helm home directory")
 	f.StringVar(&env.TillerNamespace, "tiller-namespace", tillerNamespace, "Tiller namespace for Helm")
+	f.Var(&env.DisabledStages, "disable", "Stages to be disabled")
```
Comment (Fsero) [inline cmd/apply.go]: can you edit README.md to explain what --dsable does (maybe with an example?)

File: pkg/landscaper/executor.go [hunk 2]
```diff
@@ -86,31 +88,47 @@ func (e *executor) Apply(desired, current Components) error {
 		return err
 	}
 
-	for _, cmp := range delete {
-		if err := e.DeleteComponent(cmp); err != nil {
-			logrus.WithFields(logrus.Fields{"error": err}).Error("DeleteComponent failed")
-			return err
+	if e.stageEnabled("delete") {
+		logrus.Infof("hola que hace")
```
Comment (Fsero) [inline pkg/landscaper/executor.go]: remove this log line?

File: cmd/apply.go (hunk 1 above)
Comment (zmalik) [reply cmd/apply.go]: yep! going for it!

**Comment by zmalik** [comment]
🙇 thanks @Fsero ! addressed the comments

**Comment by ivanilves** [comment]
:clap:

**Pull request merged**